workflow.swift

```diff
@@ -56,14 +69,15 @@ wait(ready) {
         printf("swift: The launched application did not succeed.");
     }
 
-    program2 = "stage_write/stage_write";
-    arguments2 = split("heat.bp staged.bp %s \"\" MPI \"\"" % rmethod , " ");
-    printf("size: %i", size(arguments2));
-    printf("swift: launching: %s", program2);
-    exit_code2 = @par=swproc launch(program2, arguments2);
-    printf("swift: received exit code: %d", exit_code2);
-    if (exit_code2 != 0)
+    if(rmethod == "BP")
     {
-        printf("swift: The launched application did not succeed.");
+        wait (exit_code1)
```

Comment:
I agree with @mw70. File-based coupling doesn't look useful for CoDAR's purpose. It can be used as a debugging purpose. In the meantime, FlexPath looks like always working solidly.